src/modelfree/train.py

```diff
@@ -142,18 +142,26 @@ def callback(locals, globals):
     model.learn(total_timesteps=total_timesteps, log_interval=1, seed=_seed, callback=callback)
     final_path = osp.join(out_dir, 'final_model')
     _save(model, final_path, save_callbacks)
-    model.sess.close()
+    if isinstance(model, GAIL):
```

Comment:
Can you instead make a PR for Stable Baselines to make the GAIL class expose `sess` from `trpo.sess`? Stable Baselines should have a consistent interface here.